index.html

```diff
@@ -2360,69 +2367,124 @@ <h2 id="OfflineAudioCompletionEvent">
+          <li>Return <var>o</var>
+          </li>
+        </ol>
+        <p>
+          To create a new <a>AudioNode</a> of a particular type <var>i</var>
+          using its <a>factory method</a>, called on a
+          <var>BaseAudioContext</var> <var>c</var>, execute these steps:
+        </p>
+        <ol>
+          <li>Let <var>o</var> be a new object of type <var>i</var>.
+          </li>
+          <li>Let <var>option</var> be a dictionnary of the type associated to
+          the interface associated to this factory method.
+          </li>
+          <li>For each parameter passed to the factory method, set the
+          dictionary member of the same name on <var>option</var> to the value
+          of this parameter.
+          </li>
+          <li>
+            <a href="#audionode-constructor-common">Initialize the common
+            part</a> of <var>o</var> with <var>c</var> and <var>option</var> as
+            arguments.
+          </li>
+          <li>Return <var>o</var>
+          </li>
+        </ol>
+        <p>
+          <dfn id="audionode-constructor-common">Initializing the common
+          part</dfn> of an object <var>o</var> of interface <var>i</var> that
+          inherits from <a>AudioNode</a> means executing the following steps,
+          given the arguments <var>context</var> and <var>dict</var> passed to
+          the constructor of this interface.
+        </p>
         <ol>
-          <li>An <a>AudioNode</a> can be created by calling the corresponding
-          <dfn>factory method</dfn> on the <a>AudioContext</a>, passing in
-          arguments. The <a>AudioNode</a> is associated with the
-          <a>AudioContext</a> it was created from.
+          <li>Set <var>o</var>'s associated <a>BaseAudioContext</a> to
+          <var>context</var>.
+          </li>
+          <li>Set its value for <a href=
+          "#widl-AudioNode-numberOfInputs">numberOfInputs</a>, <a href=
+          "#widl-AudioNode-numberOfOutputs">numberOfOutputs</a>, <a href=
+          "#widl-AudioNode-channelCount">channelCount</a>, <a href=
+          "#widl-AudioNode-channelCountMode">channelCountMode</a>, <a href=
+          "#widl-AudioNode-channelInterpretation">channelInterpretation</a> to
+          the default value for this specific interface outlined in the section
+          for each <a>AudioNode</a>.
+          </li>
+          <li>If the <a>AudioNode</a> being constructed is a
+          <a>ConvolverNode</a>, set its <a href=
+          "#widl-ConvolverNode-normalize">normalize</a> attribute with the
+          inverse of the value of the <a href=
+          "#widl-ConvolverOptions-disableNormalization">disableNormalization</a>
+          in <var>dict</var>, and then set its <a href=
+          "#widl-ConvolverNode-buffer">buffer</a> attribute to the value of the
+          <a href="#widl-ConvolverOptions-buffer">buffer</a> in <var>dict</var>
+          member, in this order, and jump to the last step of this algorithm.
```

Comment:
Yes, this algorithm describes a way of getting the desired result. There are other ways to get the same result.